apps/patient/src/views/Status.tsx

```diff
@@ -15,6 +26,67 @@ import { formatAddress, getFulfillmentType, preparePharmacy } from '../utils/gen
 import { orderStateMapping as m, text as t } from '../utils/text';
 import { useOrderContext } from './Main';
 import { HorizontalStatusStepper } from '../components/HorizontalStatusStepper';
+import dayjs from 'dayjs';
+
+function formatReadyTime(baseTime: string, minutesUntilReady: number): ReactElement {
```

Comment:
This should be a react component not a function. 

Use like 
```
<ReadyForPickup baseTime={createdAt} minutesUntilReady={minutesUntilReady} />
```

As currently set up, it'll rerender every time, whereas if its a component, React will only render it when props change.

Reply:
pretty much, as a function, this will rerender every time because it won't be part of the shadow DOM. The code isn't bad performing, but its not ideal.

If its a component, the code WILL be part of the shadow DOM, so it'll only rerender when the parent rerenders (never) or the props change (also never).